realm/realm-library/src/main/java/io/realm/internal/UncheckedRow.java

```diff
@@ -188,6 +188,13 @@ public LinkView getLinkList(long columnIndex) {
         return new LinkView(context, parent, columnIndex, nativeLinkViewPtr);
     }
 
+    @Override
+    public void checkIfBacklinkAvailable() {
```

Comment:
The main concern now is whether we can provide a backlinks object to users, but yes, that's the same with whether it's attached or not. And latter is more versatile. 

How about `checkIfAttached()`?